qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +135,259 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
+        """Return the default and up to date instruction schedule map."""
+        return self._inst_map
+
+    def _get_operated_qubits(self) -> Dict[int, List[int]]:
+        """Get a dict describing qubit couplings.
+
+        This is an extension of the coupling map and used as a convenience to help populate
+        the instruction schedule map.
+
+        Returns:
+            A dict where the key is the number of qubits coupled and the value is a list of
+            lists where the sublist shows which qubits are coupled. For example, a three qubit
+            system with a three qubit gate and three two-qubit gates would be represented as
+
+            .. parsed-literal::
+
+                {
+                    1: [[0], [1], [2]],
+                    2: [[0, 1], [1, 2], [2, 1]],
+                    3: [[0, 1, 2]]
+                }
+        """
+        operated_qubits = defaultdict(list)
+
+        # Single qubits
+        for qubit in self._qubits:
+            operated_qubits[1].append([qubit])
+
+        # Multi-qubit couplings
+        if self._coupling_map is not None:
+            for coupling in self._coupling_map:
+                operated_qubits[len(coupling)].append(coupling)
+
+        return operated_qubits
+
+    def get_inst_map(
+        self,
+        group: str = "default",
+        cutoff_date: datetime = None,
+    ) -> InstructionScheduleMap:
+        """Get a new instance of an Instruction schedule map.
+
+        Args:
+            group: The calibration group from which to draw the parameters.
+                If not specified this defaults to the 'default' group.
+            cutoff_date: Retrieve the most recent parameter up until the cutoff date. Parameters
+                generated after the cutoff date will be ignored. If the cutoff_date is None then
+                all parameters are considered. This allows users to discard more recent values that
+                may be erroneous.
+
+        Returns:
+            An instruction schedule map with parameters updated up to the desired cutoff date
+            and from the desired calibration group.
+        """
+        inst_map = InstructionScheduleMap()
+
+        self.update_inst_map(group=group, cutoff_date=cutoff_date, inst_map=inst_map)
+
+        return inst_map
+
+    def update_inst_map(
+        self,
+        schedules: Optional[set] = None,
+        qubits: Optional[Tuple[int]] = None,
+        group: Optional[str] = "default",
+        cutoff_date: datetime = None,
+        inst_map: Optional[InstructionScheduleMap] = None,
+    ):
+        """Push all schedules from the Calibrations to the inst map.
+
+        This will create instructions with the same name as the schedules.
+
+        Args:
+            schedules: The name of the schedules to update. If None is given then
+                all schedules will be pushed to instructions.
+            qubits: The qubits for which to update the instruction schedule map.
```

Comment:
I would clarify in the docstring here that `qubits` specifies a particular set of qubits to add instructions for. A user might think that specifying `(1, 2, 3)` would update all the single and two qubit gates for qubits 1, 2, and 3 rather than just updating any schedules using all three of those qubits (likely none).

Reply:
Done here: 4cea743